### ISSUE TITLE
Consider creating video tutorial for deployment

### ISSUE THREAD

**Issue by darshitac11** [Atri-Labs/framework-docs#63]
https://atricommunity.slack.com/archives/C03TJPYR2E7/p1666858695622259

**Comment by GauravBurande** [comment]
https://youtube.com/playlist?list=PLEtjGa9VEukNoeX7jK4pz9QMLwXqltazo

Can we add this in the documentation?

**Comment by GauravBurande** [comment]
It's full A-Z on how to everything about atri framework

**Comment by darshitac11** [comment]
Yes, it's a good tutorial series by the community. Glad that you found it helpful. 

However, we are releasing v1.0 which will change some of the steps mentioned in these tutorials.
